Sources/RInAppMessaging/Views/Presenters/FullViewPresenter.swift

```diff
@@ -38,14 +41,15 @@ internal class FullViewPresenter: BaseViewPresenter, FullViewPresenterType {
 
         var buttonsToAdd = [(ActionButton, ActionButtonViewModel)]()
         for (index, button) in supportedButtons.enumerated() {
+            let backgroundColor = UIColor(hexString: button.buttonBackgroundColor) ?? .white
             buttonsToAdd.append((
                 ActionButton(impression: index == 0 ? ImpressionType.actionOne : ImpressionType.actionTwo,
                              uri: button.buttonBehavior.uri,
                              trigger: button.campaignTrigger),
                 ActionButtonViewModel(text: button.buttonText,
                                       textColor: UIColor(hexString: button.buttonTextColor) ?? .black,
-                                      backgroundColor: UIColor(hexString: button.buttonBackgroundColor) ?? .white)))
-
+                                      backgroundColor: UIColor(hexString: button.buttonBackgroundColor) ?? .white,
+                                      shouldDrawBorder: viewBackgroundColor == backgroundColor)))
```

Comment:
This means the colours have be **exactly** the same? What if we have a white on off-white? Seems like there should be a degree of tolerance here

Reply:
@apparition47 PDM confirmed that there should be some tolerance. You win 😉

Reply:
Oh, shouldn't have asked him then 😅